src/components/TeamPage/index.jsx

```diff
@@ -0,0 +1,117 @@
+import React from 'react';
+import Header from '../Header';
+import LandingFooter from '../LandingFooter';
+import './TeamPage.css';
+import MemberCard from '../MemberCard';
+import Engineer from '../../assets/images/Engineer.png';
+import Alex from '../../assets/images/Alex.png';
+import Aminah from '../../assets/images/Aminah.png';
+import Greatest from '../../assets/images/Greatest.png';
+import Allan from '../../assets/images/Allan.png';
+import Henry from '../../assets/images/Henry.png';
+import Dorothy from '../../assets/images/Dorothy.png';
+import Steve from '../../assets/images/Steve.png';
+import Paul from '../../assets/images/Paul.png';
+import Colin from '../../assets/images/Colin.png';
+import Mary from '../../assets/images/Mary.png';
+import Rajab from '../../assets/images/Rajab.png';
+
+class TeamPage extends React.Component {
+
+  render() {
+    return (
+      <div className="TeamPageContainer">
+        <Header />
+        <div className="TeamImagesRow">
```

Comment:
since u explicitly asked for my review, will do what i had avoided earlier: do not statically render each member card - have a list of the members (json maybe) and iterate over that

Reply:
let me know if that makes sense

Reply:
yeah +1 on @colinwagaba's request, @neelxie you can do that... otherwise well done so far

Reply:
@neelxie a slight change in my name. It should be Naggita .

Reply:
@colinwagaba I thought of that prior for reusability purposes but its not applicable in my case as a JSON does not provide an ordered outcome like an array, thus we will have a jumbled list that looks very disorganized. I've tried it.

Reply:
I have used a different but effective method, check it out

Reply:
looks effective

Reply:
ok for now